docs/index.rst

```diff
@@ -4,7 +4,8 @@
 SkyPy Documentation
 ###################
 
-This package contains methods for modelling the Universe, galaxies and the Milky Way. Also included are methods for generating observed data.
+This package contains methods for modelling the Universe, galaxies and the
+Milky Way. Also included are methods for generating observed data.
```

Comment:
This package contains methods for modelling the Universe, galaxies and the Milky Way. Also included are methods for generating observed data.